### ISSUE TITLE
Leaving a month name empty causes NullPointerException when calculating leapYear

### ISSUE THREAD

**Issue by thatvineyard** [Anrik21/Fantasymanager#6]
On the empty month `name` is null, causing a NEP in `leapYearMonth1.Items.Add(name)`. 

Could be fixed by creating a default empty name or by preventing progression unless months are named. 

**GeneratorForm.cs - Line 317-321**
```
            foreach (string name in ourCalendar.MonthNames)
            {
                leapYearMonth1.Items.Add(name);
                hdayMonthSelector1.Items.Add(name);
            }
```

**Comment by Anrik21** [comment]
Will add a "empty" month name so that it'll go through & make sure to build remaining GUI to treat "empty" as it's month numeral instead.

**Comment by Anrik21** [comment]
Fixed in commit fcf5ae6f9ba1d1b6cab7cab8f37c7d92261f6651
